packages/jamf_pro/data_stream/events/elasticsearch/ingest_pipeline/default.yml

```
@@ -67,44 +70,47 @@ processors:
 - date:
     field: jamf_pro.events.event.last_update
     target_field: jamf_pro.events.event.last_update
-    if: "ctx.jamf_pro.events?.last_update != null"
+    if: "ctx.jamf_pro.events.event.last_update != null"
```

```suggestion
    if: ctx.jamf_pro?.events?.event?.last_update != null
```
Unless we're confident in the existence of the `ctx.jamf_pro.events.event` path. If we are, we should probably have a comment at the point from which we guarantee it exists. (similar below)

---

Yes, we're confident. Did it as above.